package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "eslint-config-typescript",
```

Comment:
We should use a scoped name and make this just `eslint-config`, as TS is a superset of JS and that name can be auto imported. On NPM, that's `@thesis-co/eslint-config`. I can follow up with a PR to do builds + releases

Reply:
If you *do* want a JS-only config and to keep this at `eslint-config-typescript`, it would be good to see what that looks like and extend it here... maybe that was the intent?

Reply:
I wanted to have a TS-only config here. I've changed it as you suggested in your first comment cbd5e2b Good point on scoping the name.

Reply:
In my other comment, I suggested moving https://github.com/keep-network/eslint-config-keep under `thesis` org. In such a case, we need to make sure the names don't clash.

> If you do want a JS-only config and to keep this at eslint-config-typescript, it would be good to see what that looks like and extend it here... maybe that was the intent?

I like this suggestion very much (`@thesis-co/eslint-config-typescript` extends `@thesis-co/eslint-config`), it could be worth checking if it's possible.